frog/frog.scrbl

```diff
@@ -924,6 +924,19 @@ This may be an absolute or relative path. If relative, it's relative
 to the project top directory, i.e. to where @secref["config"] is
 located.}
 
+@defparam[current-rebuild?
```

Comment:
Although I don't necessarily object to this change, I don't understand the use case? Typically I use `--watch` because I want things to rebuild. Wouldn't I simply not use `--watch` if/when I don't want that?

Reply:
Actually it's the other way around: I want some `html` change to trigger a rebuild. Currently, it won't because all `html`s are blacklisted.

Reply:
A more radical question is, why having a blacklist at all? The watch is done only in the `_src` directory, but as I understand, the output html/xml/txt files would be outside of `_src`.

Reply:
So, if you think that there should be a blacklist, this patch simply gives you a finer control of that list, and you can merge it if everything looks good. On the other hand, if you think blacklist is not a good idea anymore, let me know and I can submit a change to remove all blacklist mechanism away.